wiki/Gameplay/Input_device/Beatmania_controller/id.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Kontroller Beatmania
+
+![](img/beatmania-controller.jpg "Kontroller Beatmania 7 tombol")
+
+**Kontroller Beatmania** adalah sebuah tipe dari kontroller gim yang cocok digunakan pada seri game *[Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia")*.
```

Comment:
```suggestion
**Kontroller Beatmania** adalah sebuah tipe kontroller yang digunakan pada seri game *[Beatmania](https://id.wikipedia.org/wiki/Beatmania "Wikipedia")*.
```

Reply:
the id page doesn't exist yet. keeping this as original now